### ISSUE TITLE
Problems installing

### ISSUE THREAD

**Issue by chavab1** [geerlingguy/drupal-vm#468]
This is the first time I'm using Vagrant. I'm following the instructions; I downloaded VirtualBox and Vagrant. I copied and made the changes to the example files. When I try to run `vagrant up` I get the following error message:

```
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["clonevm", "24ec6ddb-7c6d-47be-ad3b-c3cfb5213756", "--register", "--name", "temp_clone_1456361698378_21314"]

Stderr: VBoxManage.exe: error: Could not find a registered machine with UUID {24ec6ddb-7c6d-47be-ad3b-c3cfb5213756}
VBoxManage.exe: error: Details: code VBOX_E_OBJECT_NOT_FOUND (0x80bb0001), component VirtualBox, interface IVirtualBox, callee IUnknow
VBoxManage.exe: error: Context: "FindMachine(Bstr(pszSrcName).raw(), srcMachine.asOutParam())" at line 434 of file VBoxManageMisc.cpp
```

I know it's not an issue with VirtualBox because I'm able to run that command from another vagrant file.

Any help would be really appreciated.

Thanx.

**Comment by oxyc** [comment]
Any chance you're computer has an `i3` CPU? 

- https://github.com/geerlingguy/drupal-vm/issues/439
- https://github.com/geerlingguy/drupal-vm/commit/532ccf77a2354ae249c00c4c9b87434e82809623

**Comment by chavab1** [comment]
Thanx for the quick reply. After a long while of looking around I finally got it to work. 

I tried to install it yesterday and it went through successfully. However, at the very last stage I got an error telling me that it couldn't connect. It turned out my VT-x was disabled. I enabled my VT-x and then ran through the setup again. That's when I started getting the current error. So, I uninstalled VirtualBox, deleted the .VirtualBox directory from my user profile directory and did a clean install. That did the trick.

So I'm thinking the first time I ran the setup when my VT-x was disabled it created the machine successfully even though it wasn't appearing in the VirtualBox console. Then after I enabled my VT-x and ran it again it was conflicting with that original machine.

I'm new to VirtualBox and Vagrant, I'm sure someone with more experience would've know right away.

Anyways, thanx!!
